### DISCUSSION TITLE
The overlay component crashing my app

### DISCUSSION THREAD

**Discussion by AmirHo3einJS** [vercel/next.js#37534] [Help] [answered]
![error](https://user-images.githubusercontent.com/101206088/172540430-c8264d93-17d6-40ab-a75c-b66510d07b1c.gif)
Hello, as I mention in the title, I don't know why this is happening and the number of errors is constantly increasing. When my app has an error this happens too.

I've installed nextjs with `npx create-next-app@latest --typescript --use-npm` and below is my package.json file:

{
  "name": "metachimps",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@emotion/react": "^11.9.0",
    "@emotion/styled": "^11.8.1",
    "@hookstate/core": "^4.0.0-rc.3",
    "@hookstate/persistence": "^3.0.0",
    "@metaplex-foundation/js-next": "^0.9.0",
    "@mui/material": "^5.8.2",
    "@solana/web3.js": "^1.43.5",
    "axios": "^0.27.2",
    "jsonwebtoken": "^8.5.1",
    "mongoose": "^6.3.5",
    "mongoose-unique-validator": "^3.1.0",
    "next": "12.1.6",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-toastify": "^9.0.3"
  },
  "devDependencies": {
    "@types/jsonwebtoken": "^8.5.8",
    "@types/mongoose-unique-validator": "^1.0.6",
    "@types/node": "17.0.40",
    "@types/react": "18.0.12",
    "@types/react-dom": "18.0.5",
    "autoprefixer": "^10.4.7",
    "eslint": "8.17.0",
    "eslint-config-next": "12.1.6",
    "postcss": "^8.4.14",
    "tailwindcss": "^3.0.24",
    "typescript": "4.7.3"
  }
}

**Comment by icyJoseph** [comment, marked as answer]
Very strange, I do see that there's a TypeError when trying to use `Symbol('self')`, which I found on the `hookstate` set of packages, still though, kinda hard to see, https://github.com/avkonst/hookstate/blob/c159e6c255b0a9795218691c4dfb78c29ffae533/src/index.ts#L710.

And look at what I found! https://github.com/avkonst/hookstate/pull/293 I think this is the issue! Looks like they have an unpublished fix already.

**Reply by AmirHo3einJS** [reply]
Thank you, you're right. it was the hookstate causing this problem when it was creating a global state. I had to uninstall it and use redux. I hope this issue is fixed and we could use hookstate with nextjs.
